packages/at_secondary_server/lib/src/verb/handler/enroll_verb_handler.dart

```diff
@@ -243,6 +252,18 @@ class EnrollVerbHandler extends AbstractVerbHandler {
     responseJson['enrollmentId'] = enrollmentIdFromParams;
   }
 
+  Future<void> _updateEnrollmentAccess(
+      String enrollId, EnrollDataStoreValue existingEnrollmentValue) async {
```

Comment:
don't pass existingEnrollmentValue into function and update..instead return new namespace list from a method and update the namespace outside the method

Reply:
addressed in [6cd81fa](https://github.com/atsign-foundation/at_server/pull/1562/commits/6cd81faef234f9eb77d2743856046f94e27720ee)